app/models/concerns/translatable.rb

```diff
@@ -2,7 +2,7 @@ module Translatable
   extend ActiveSupport::Concern
 
   included do
-    has_many :translations, as: :translatable
+    has_many :translations, as: :translatable, autosave: true
```

Comment:
With the autosave enabled and the other change to make sure the Translation instance which is included in the association list is updated when the field setter is used, things now behave as I believe is desired:  The change remains transient until the parent record is saved, and then the dependent change is persisted automatically with the parent.  
Without both changes, a translatable field was only getting saved with the initial record creations, but changes to those fields were never getting saved.